### ISSUE TITLE
ResembleDepmap crashing due to bug in code

### ISSUE THREAD

**Issue by devianggr** [WubingZhang/MAGeCKFlute#26]
Hello, 

Thank you for providing a nice and easy-to-work package!

While following the vignette documentation: https://www.bioconductor.org/packages/release/bioc/vignettes/MAGeCKFlute/inst/doc/MAGeCKFlute.html#section-ii-downstream-analysis-of-mageck-rra 
I had an error in section 2.3.1 with the ResembleDepmap function.
![image](https://github.com/user-attachments/assets/ded07b0e-9168-4164-b33c-d8ba838cb4e6)
After playing around with different scenarios, I found out that the issue stems from the ResembleDepmap, specifically in: rownames(similarity) = sampleinfo[colnames(Depmap), 1]
I also tried narrowing down the lineage to one but ran into a different issue: 
![image](https://github.com/user-attachments/assets/2ce7f3d8-304b-4745-9643-f8e7410fbe63)
Then, i found out that the issue is that the column cell_line_name from the depmap metadata loaded by the MAGeCKFlute package contains duplicated cell line names and missing values.
Therefore, I created and sourced an edited version of the 'IncorporateDepmap.R' file where I changed your above line of code to: 
rownames(similarity) = sampleinfo[colnames(Depmap), 'cell_line']
and ran it only for one lineage ("Pancreatic Cancer") and it worked. 

In summary, the code should be edited so as to account for the value missingness/duplicated in the metadata of depmap loaded in your package. 

Thank you in advance.
